config/blast.php

```diff
@@ -3,9 +3,47 @@
 return [
     'storybook_server_url' =>
         env('STORYBOOK_SERVER_HOST', env('APP_URL')) . '/storybook_preview',
+    
+    /**
+     * See https://storybook.js.org/docs/react/essentials/controls Set
+     * to true to enable full documentation on the controls tab. 
+     */
+    'storybook_expanded_controls' => true,
 
-    // See https://storybook.js.org/docs/react/configure/theming for available options
-    'storybook_theme' => [],
+    /**
+     * See https://storybook.js.org/docs/react/configure/theming for
+     * detail - dark, normal, custom. Dark and normal are out of
+     * the box from @storybook-theming. For custom edit the
+     * values in 'custom_theme' to create a custom theme
+     */
+    'storybook_theme' => 'custom',
+
+    'storybook_custom_theme' => [
+        'base' => 'lightgray',
+        'colorPrimary' => 'cadetblue',
+        'colorSecondary' => 'lightcoral',
+        'appBg' => 'whitesmoke',
+        'appContentBg' => 'gainsboro',
+        'appBorderColor' => 'ghostwhite',
+        'appBorderRadius' => 4,
+        'fontBase' => '"Nunito", sans-serif',
+        'fontCode' => 'monospace',
+        'textColor' => 'darkslategrey',
+        'textInverseColor' => 'darkgrey',
+        'barTextColor' => 'white',
+        'barSelectedColor' => 'ghostwhite',
+        'barBg' => 'cadetblue',
+        'inputBg' => 'ghostwhite',
+        'inputBorder' => 'ghostwhite',
+        'inputTextColor' => 'darkslategrey',
+        'inputBorderRadius' => 4,
+        'brandTitle' => 'Blast - Storybook for blade',
+        'brandUrl' => '#',
+        'brandImage' => 'https://i.gifer.com/nN2.gif'
+    ],
+
+    // dark or normal
+    'storybook_docs_theme' => 'custom',
```

Comment:
Can this be moved above the `storybook_custom_theme` array to make it easier to see? I missed it when testing this out.